src/components/DocumentationTopic.vue

```diff
@@ -426,6 +439,14 @@ export default {
       || (downloadNotAvailableSummary && downloadNotAvailableSummary.length)
       || (primaryContentSections && primaryContentSections.length)
     ),
+    viewMoreLink: ({
+      references,
+      identifier,
+      objcPath,
+      swiftPath,
+    }) => (
+      references[identifier] ? references[identifier].url : (objcPath || swiftPath)
```

Comment:
If this is meant to be a link to the page itself, I'm not sure it makes sense to look in `references`, because there would only be one there if it referenced itself.

I think your default case makes sense as the only way to reliably get the link to the current page (`(objcPath || swiftPath)`) but it seems like it would always favor the ObjC variant if one exists, regardless of what the current preference is—we might want to tweak that.

Reply:
Right, we should try and check the `this.interfaceLanguage` and fallback to whatever is available. 

Is there a chance that one is not available, when its language is picked? Probably not... 🤔 

```suggestion
      interfaceLanguage === Language.objectiveC.key.api ? objcPath : swiftPath
```